airflow/providers/cncf/kubernetes/executors/kubernetes_executor_utils.py

```diff
@@ -250,10 +249,18 @@ def process_status(
                                 and container_status_state["waiting"]["message"] == "pull QPS exceeded"
                             ):
                                 continue
+                            self.log.error(
+                                "Event: %s has container %s with fatal reason %s",
```

Comment:
Will it always be a fatal reason when it reaches this place in the code? How about the waiting reason `ContainerCreating`?